index.js

```diff
@@ -10,31 +10,26 @@ fs.readdir('./frames').then(data => {
   data.forEach(async frame => {
     const f = await fs.readFile(`./frames/${frame}`);
     frames.push(f.toString());
-  })
+  });
 });
 
-const colorsOptions = ['red', 'yellow', 'green', 'blue', 'magenta', 'cyan', 'white'];
-const numColors = colorsOptions.length;
+const selectColor = function(previousColor) {
+  const colors = ['red', 'yellow', 'green', 'blue', 'magenta', 'cyan', 'white'];
```

Comment:
Why create this variable each time this function is run? I don't see the benefit of moving it inside the function.